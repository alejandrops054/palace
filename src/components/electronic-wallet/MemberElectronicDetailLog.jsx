/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from "react";
import { Table } from "antd";
import { Input, Select, Button, Hr } from "../index";

import "antd/dist/antd.css";

const MemberElectronicDetailLog = () => {
  const columns = [
    {
      title: "User ",
      dataIndex: "user",
      key: "user",
      width: 100,
    },

    {
      title: "Date/Time",
      dataIndex: "date",
      key: "date",
      width: 100,
      render: () => (
        <Button
          col="1.5"
          icon="fas fa-book"
          typebootstrap="link"
          type="button"
        />
      ),
    },
    {
      title: "Module ",
      dataIndex: "module",
      key: "module",
      width: 100,
    },
    {
      title: "Action ",
      dataIndex: "action",
      key: "action",
      width: 100,
    },
    {
      title: "Original ",
      dataIndex: "original",
      key: "original",
      width: 150,
    },
    {
      title: "Changes ",
      dataIndex: "changes",
      key: "changes",
      width: 150,
    },
  ];
  const data = [];
  return (
    <div className="container mt-3">
      <div className="row">
        <div className="col-md-3">
          <Input type="date" label="Date From " placeholder="" col={12} />
        </div>
        <div className="col-md-3"></div>
        <div className="col-md-3">
          <Input type="text" label="Field Name" placeholder="" col={12} />
        </div>
        <div className="col-md-3">
          <Button
            title="Apply Filter"
            col="12"
            icon="fas fa-filter"
            typebootstrap="link main"
            type="button"
          />
        </div>
      </div>
      <div className="row">
        <div className="col-md-3">
          <Input type="date" label="Date To " placeholder="" col={12} />
        </div>
        <div className="col-md-3">
          <Input type="text" label="User To" placeholder="" col={12} />
        </div>
        <div className="col-md-3">
          <Input type="text" label="Value" placeholder="" col={12} />
        </div>
        <div className="col-md-3">
          <Button
            title="Clear Filter"
            col="12"
            icon="fas fa-filter"
            typebootstrap="link main"
            type="button"
          />
        </div>
      </div>
      <div className="row">
        <Table
          columns={columns}
          dataSource={data}
        />
      </div>
      <br/>
      <div className="row">
      <Button title="Close" col={3} typebootstrap="secondary" type="button" />
      </div>
    </div>
  );
};
export default MemberElectronicDetailLog;
