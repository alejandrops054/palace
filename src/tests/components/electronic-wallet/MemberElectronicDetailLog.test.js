/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import MemberElectronicDetailLog from '../../../components/electronic-wallet/MemberElectronicDetailLog'

describe('MemberElectronicDetailLog component', () => {
  test('debe mostrar snapshot', () => {

    const wrapper = shallow(<MemberElectronicDetailLog />)
    expect(wrapper).toMatchSnapshot();
  })

})