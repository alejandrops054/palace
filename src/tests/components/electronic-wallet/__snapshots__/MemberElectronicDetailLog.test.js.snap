// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MemberElectronicDetailLog component debe mostrar snapshot 1`] = `
<div
  className="container mt-3"
>
  <div
    className="row"
  >
    <div
      className="col-md-3"
    >
      <Input
        col={12}
        label="Date From "
        placeholder=""
        type="date"
      />
    </div>
    <div
      className="col-md-3"
    />
    <div
      className="col-md-3"
    >
      <Input
        col={12}
        label="Field Name"
        placeholder=""
        type="text"
      />
    </div>
    <div
      className="col-md-3"
    >
      <Button
        col="12"
        icon="fas fa-filter"
        title="Apply Filter"
        type="button"
        typebootstrap="link main"
      />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-md-3"
    >
      <Input
        col={12}
        label="Date To "
        placeholder=""
        type="date"
      />
    </div>
    <div
      className="col-md-3"
    >
      <Input
        col={12}
        label="User To"
        placeholder=""
        type="text"
      />
    </div>
    <div
      className="col-md-3"
    >
      <Input
        col={12}
        label="Value"
        placeholder=""
        type="text"
      />
    </div>
    <div
      className="col-md-3"
    >
      <Button
        col="12"
        icon="fas fa-filter"
        title="Clear Filter"
        type="button"
        typebootstrap="link main"
      />
    </div>
  </div>
  <div
    className="row"
  >
    <Table
      columns={
        Array [
          Object {
            "dataIndex": "user",
            "key": "user",
            "title": "User ",
            "width": 100,
          },
          Object {
            "dataIndex": "date",
            "key": "date",
            "render": [Function],
            "title": "Date/Time",
            "width": 100,
          },
          Object {
            "dataIndex": "module",
            "key": "module",
            "title": "Module ",
            "width": 100,
          },
          Object {
            "dataIndex": "action",
            "key": "action",
            "title": "Action ",
            "width": 100,
          },
          Object {
            "dataIndex": "original",
            "key": "original",
            "title": "Original ",
            "width": 150,
          },
          Object {
            "dataIndex": "changes",
            "key": "changes",
            "title": "Changes ",
            "width": 150,
          },
        ]
      }
      dataSource={Array []}
      rowKey="key"
    />
  </div>
  <br />
  <div
    className="row"
  >
    <Button
      col={3}
      title="Close"
      type="button"
      typebootstrap="secondary"
    />
  </div>
</div>
`;
